src/Compiler/ReleaseManager.cs

```diff
@@ -43,25 +43,24 @@ public ReleaseManager(GitHubClient gitHubClient, string organization)
 
                     if (release != null)
                     {
-
                         var releaseUpdatedAt = GetUpdatedAt(release).ToUniversalTime();
 
                         var allIssues = await gitHubClient.AllIssuesForMilestone(milestone);
 
-                        var latestIssueModification =
-                            allIssues.Where(i => i.State == ItemState.Closed).Max(i => i.ClosedAt.Value).UtcDateTime;
-
-                        Console.Out.WriteLine("Release exists for milestone {0} - Last updated at: {1}, Issues updated at:{2}", potentialRelease, releaseUpdatedAt, latestIssueModification);
-
-                        if (releaseUpdatedAt < latestIssueModification)
+                        var allClosedIssues = allIssues.Where(i => i.State == ItemState.Closed).ToList();
```

Comment:
this will get gradually slower over time. better to do a sorted query so we dont need the `.max`